dashboard/app/views/layouts/application.html.haml

```diff
@@ -83,7 +83,7 @@
         - unless view_options[:no_footer] || view_options[:small_footer]
           .push
 
-      - if view_options[:small_footer]
+      - if view_options[:small_footer] && !view_options[:no_footer]
```

Comment:
This is a great catch -  I don't actually know. I think I probably did some not great programming here - the safest thing for me to do here is to explicitly set small_footer to false rather than anding these two things together.

Reply:
You should be safe to remove the ` && !view_options[:no_footer]` since you set small_footer to false